### PR TITLE
Workouts teacher

### PR DESCRIPTION
**Pull request by Anajuliarlc**

### CODE REVIEW

File: src/gui/frame.py [hunk 1]
```diff
@@ -50,7 +50,7 @@ def destroy(self) -> None:
 class FrameNotFound(ValueError):
     def __init__(self) -> None:
         """Raise when the frame type is not implemented"""        
-        super().__init__("Login frame not found")
+        super().__init__("Frame not found")
```
Comment (DutraIRS) [inline src/gui/frame.py:53]: Boa!

File: src/student/goals_frame.py [hunk 2]
```diff
@@ -34,27 +34,25 @@ def place_objects(self) -> None:
         
         def view_goals() -> None:
             self.destroy()
-            self.window.change_frame(sff.StudentFrameFactory.get_frame("ViewGoalsFrame", self.window))
```
Comment (DutraIRS) [inline src/student/goals_frame.py:37]: Exatamente

File: src/teacher/teacher_frame_factory.py [hunk 3]
```diff
@@ -33,12 +36,20 @@ def get_frame(type_: str, window: tk.Tk, height: int = 600, width: int = 800,
...
         elif type_ == "ClassesFrame":
             return tcf.ClassesFrame(window)
         elif type_ == "NewClass":
             return tnc.NewClassFrame(window)
+        elif type_ == "WorkoutsFrame":
+            return tw.InitialWorkoutsFrame(window)
+        elif type_ == "CurrentWorkoutsFrame":
+            return cw.CurrentWorkouts(window)
+        elif type_ == "WorkoutsFrame":
+            return tw.InitialWorkoutsFrame(window)
```
Comment (DutraIRS) [inline src/teacher/teacher_frame_factory.py:52]: bonito

**Review by DutraIRS — APPROVED** [review]

**Review by georgedutra — APPROVED** [review]
Belos workouts

**Pull request merged**